### PR TITLE
add a generic integrator_size.H

### PR DESCRIPTION
**Pull request by zingale**
this defines the size of the ODE system we integrate and knows
about the different coupling methods.

### CODE REVIEW

**Comment by zingale** [comment]
tests are good: http://groot.astro.sunysb.edu/Microphysics/test-suite/gfortran/2022-03-08-002/index.html
the sdc failure is because I hadn't merged the latest development in.

**Comment by maxpkatz** [comment]
Just for future-proofing and consistency with other bits of Microphysics, can we call this `integrator_data.H`?

**Comment by zingale** [comment]
renamed

File: integration/integrator_data.H [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#ifndef INTEGRATOR_SIZE_H
+#define INTEGRATOR_SIZE_H
```
Comment (maxpkatz) [inline integration/integrator_data.H:2]: Can you also update the header guard

Comment (zingale) [reply integration/integrator_data.H]: done

**Review by maxpkatz — APPROVED** [review]

**Pull request merged**